### PR TITLE
use json instead of simplejson

### PR DESCRIPTION
**Pull request by dannyroberts**
One fewer awkward question to answer for a new developer joining the team :)

### CODE REVIEW

File: corehq/apps/reports/tests/test_pillows_cases.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 from django.test import TestCase
-import simplejson
+import json
```
Comment (dimagimon) [inline corehq/apps/reports/tests/test_pillows_cases.py]: F401 'json' imported but unused

File: corehq/apps/reports/tests/test_pillows_xforms.py [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import copy
 from django.test import TestCase
 from django.conf import settings
-import simplejson
+import json
```
Comment (dimagimon) [inline corehq/apps/reports/tests/test_pillows_xforms.py]: F401 'json' imported but unused

File: custom/_legacy/pact/api.py [hunk 3]
```diff
@@ -274,18 +274,18 @@ def get_all_providers(invalidate=False):
         #requery and cache
         pact_hp_group = Group.by_name(PACT_DOMAIN, PACT_HP_GROUPNAME)
         providers = FixtureDataItem.by_group(pact_hp_group)
-        cache.set(PACT_PROVIDERS_FIXTURE_CACHE_KEY, simplejson.dumps([x.to_json() for x in providers]))
+        cache.set(PACT_PROVIDERS_FIXTURE_CACHE_KEY, json.dumps([x.to_json() for x in providers]))
         return providers
     else:
         try:
-            json_data= simplejson.loads(raw_cached_fixtures)
+            json_data= json.loads(raw_cached_fixtures)
```
Comment (dimagimon) [inline custom/_legacy/pact/api.py]: E225 missing whitespace around operator

**Comment by benrudolph** [comment]
just curious what the benefit is to using `json` over `simplejson`? i always used `simplejson` cause of better error msgs and it's supposedly faster

**Comment by dmyung** [comment]
I used to be an ass about choosing `simplejson` over `json` - looks like since 2.7, it's more or less a wash now.

http://stackoverflow.com/a/14513180

**Comment by dannyroberts** [comment]
Haha nice to hear you say that @dmyung :)

@benrudolph [json is simplejson](http://stackoverflow.com/a/712799/240553), the only differences are:

- `json` is in the standard library and `simplejson` is a dependency
- `simplejson` is going to be the latest head of the simplejson project, whereas `json` is going to be pinned to a specific version

Since (according to a statistic I'm making up) we use `json` in 95% of our code, it feels weird to then use `simplejson` here and there for historical reasons, and as Dan pointed out, the speed difference gained from any new commits to the `simplejson` project are pretty minimal/nonexistent at this point.

**Comment by benrudolph** [comment]
Got it, makes sense

**Comment by benrudolph** [comment]
cool, looks good to merge, except for maybe the dimagimon errors

**Pull request merged**